src/components/shared/ContentItem/ListTemplate.js

```diff
@@ -40,7 +40,7 @@ const ListTemplate = ({
     if (cuInfoRef.current && (cuInfoRef.current.scrollHeight > cuInfoRef.current.clientHeight)) {
       setIsNeedTooltip(true);
     }
-  }, [cuInfoRef.current]);
+  }, []);
```

Comment:
why you remove it? I need check if text is overfill the block, so I check it after render